File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.py
class Solution:
    def differenceOfSum(self, nums: List[int]) -> int:
        ele_sum = 0
        dig_sum = 0
        for i in range(len(nums)):
            ele_sum = ele_sum + nums[i]
        
        for j in range(len(nums)):
            while nums[j]>0:
                dig = nums[j]%10
                dig_sum = dig_sum+dig
                nums[j] = nums[j]//10
        
            
        return ele_sum - dig_sum
        

File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.cpp
class Solution {
public:
    int differenceOfSum(vector<int>& nums) {
        int ele_sum = 0, dig_sum = 0,dig,i;
        for(i=0; i<nums.size(); i++){
            ele_sum = ele_sum + nums[i];
        
            int j = nums[i];
            while(j>0){
                dig = j%10;
                dig_sum = dig_sum + dig;
                j = j/10;
            }
        }
        
        return ele_sum - dig_sum;
        
    }
};